.prerequisiteversions

```diff
@@ -0,0 +1 @@
+CMake=3.7.2
```

Comment:
I can live with a different file name for transitioning but I obviously want these to merge together at some point.